.gitignore

```diff
@@ -34,3 +34,6 @@ deploy
 /monitor/monitor-tor/*
 .java-version
 .localnet
+scripts/*.tar.gz
```

Comment:
The binaries are downloaded to .localnet, so there's no need to add these ignores.